From 2a5f04ddbe46ef378b2f168a92bbc744190ed41f Mon Sep 17 00:00:00 2001
From: Tao Ren <taoren@meta.com>
Date: Mon, 12 Feb 2024 14:09:50 -0800
Subject: [PATCH] ARM: dts: aspeed: Enable jtag1 in fblite-r1 dts

Summary:

Enable jtag1 in fblite-r1 device tree for firmware upgrade.

Test Plan:

1) "bitbake janga-image" succeeded.
2) "bitbake tahan-image" succeeded.
3) verified jtag is enabled after booting up janga and tahan.
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-fblite-r1.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-fblite-r1.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-fblite-r1.dts
index 4264bd3c7a6b..dee1f78b8ca0 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-fblite-r1.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-fblite-r1.dts
@@ -92,3 +92,7 @@ flash1@0 {
 		};
 	};
 };
+
+&jtag1 {
+	status = "okay";
+};
-- 
2.43.0

